jest.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   ],
   "roots": [
     "<rootDir>/test",
+    "<rootDir>/docs/test",
```

Comment:
I think this can go into the normal test directory.  `.ts` files are only run in node, not with mocha in the browser.  See `test/types/peg.test-d.ts` for an example.  They should "just work" with ts-node, iirc.